src/dmd/mtype.d

```diff
@@ -5220,6 +5224,35 @@ extern (C++) final class TypeDelegate : TypeNext
     }
 }
 
+/**
+ */
+extern (C++) final class TypeTraits : Type
```

Comment:
Please add some documentation for this class : why is it needed, when it is created and what information is kept in each field. Hope that is the intent with the empty comments :+1:

Reply:
Indeed i planned to do it but forgot to.